dags/longitudinal.py

```diff
@@ -59,8 +60,11 @@
     job_name="Cross Sectional View",
     execution_timeout=timedelta(hours=10),
     instance_count=30,
-    env={"date": DS_WEEKLY, "bucket": "{{ task.__class__.private_output_bucket }}"},
-    uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/cross_sectional_view.sh",
+    env = tbv_envvar("com.mozilla.telemetry.views.CrossSectionalView", {
+        "executor-cores": 8,
```

Comment:
@fbertsch is this option even needed? I don't see why it would be desirable to use less than the amount of parallelism available.

Reply:
@harterrt  I think you're the appropriate one to tag here

Reply:
also used in [jobs/focus_event_longitudinal_view.sh#L13](https://github.com/mozilla/telemetry-airflow/blob/master/jobs/focus_event_longitudinal_view.sh#L13)

Reply:
@harterrt bumping

Aside from that, I think this option should be moved into the `other` argument as opposed to the jobs argument.

Reply:
Sorry - I started a review and never submitted it.  

This is probably copy-pasta from the Longitudinal job, so I'm not attached to it. I certainly can't argue that it's necessary.

Reply:
okay, dropped it